### PR TITLE
show warn and do not fetch unnecesary data when prometheus/alertmanager base url is not available

### PR DESCRIPTION
**Pull request by suomiy**

### CODE REVIEW

File: frontend/public/kubevirt/components/cluster/cluster-overview.jsx [hunk 1]
```diff
@@ -167,27 +169,37 @@ export class ClusterOverview extends React.Component {
...
 
-    this.fetchPrometheusQuery(CEPH_OSD_UP_QUERY, 'cephOsdUp');
-    this.fetchPrometheusQuery(CEPH_OSD_DOWN_QUERY, 'cephOsdDown');
+      this.getStorageMetrics();
 
-    this.getStorageMetrics();
+      this.fetchPrometheusQuery(UTILIZATION_CPU_USED_QUERY, 'cpuUtilization');
+      this.fetchPrometheusQuery(UTILIZATION_MEMORY_USED_QUERY, 'memoryUtilization');
+    } else {
+      warn('Prometheus BASE URL is missing!');
```
Comment (rawagner) [inline frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: prometheus results will remain undefined which will result in loading states for cards which expect them. I'd expect that the cards will indicate not available state

Comment (suomiy) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: yup, my bad! I will take a look at it.

Comment (suomiy) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: fixed. The cards will show not available now. Maybe we should ask designers to show that Prometheus/AlertManager  is available in the health card as a followup?

File: frontend/public/kubevirt/components/cluster/cluster-overview.jsx [hunk 2]
```diff
@@ -137,39 +179,45 @@ export class ClusterOverview extends React.Component {
...
-    fetchPrometheusQuery(NODE_CONSUMERS_MEMORY_QUERY, result => this.onFetch('infraMemoryResults', result));
-    fetchPrometheusQuery(NODE_CONSUMERS_CPU_QUERY, result => this.onFetch('infraCpuResults', result));
+    if (getPrometheusBaseURL()){
+      fetchPrometheusQuery(CEPH_STATUS_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.cephHealth, result));
+      fetchPrometheusQuery(CONSUMERS_CPU_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadCpuResults, result));
+      fetchPrometheusQuery(CONSUMERS_MEMORY_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadMemoryResults, result));
+      fetchPrometheusQuery(CONSUMERS_STORAGE_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadStorageResults, result));
+      fetchPrometheusQuery(CONSUMERS_NETWORK_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadNetworkResults, result));
 
-    fetchPrometheusQuery(NODE_CONSUMERS_STORAGE_QUERY, 'infraStorageResults');
```
Comment (suomiy) [inline frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: how did this work before (#361) ? This does not seem to not call onFetch when the query finishes. + other few queries as well.

Comment (suomiy) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: @rawagner

Comment (rawagner) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: that is error in rebasing. This PR added those queries https://github.com/kubevirt/web-ui/pull/351 but after merging I didnt properly rebased #361. Thanks for catching it!

File: frontend/public/kubevirt/components/cluster/cluster-overview.jsx (hunk 1 above)
Comment (lizsurette) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: @suomiy - You mean work in a way to tell the user that Prometheus is unavailable, right?

What do you think @andybraren?

Comment (suomiy) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: yes, + I guess also AlertManager could be unavailable - we might also work better on the detection

Comment (andybraren) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: We have an upcoming story to reassess how the Health card works, so this is good timing. Is something like this roughly what you’d expect to see?

health-prometheus

Comment (suomiy) [reply frontend/public/kubevirt/components/cluster/cluster-overview.jsx]: +1 looks good. 

We might also distinguish between Prometheus and AlertManager - both belong to monitoring. 
And between states not available and down (the pods for each on of them could crash).

**Pull request merged**